README.md

```diff
@@ -66,9 +66,10 @@ Tests requires valid Keboola Management API tokens and an endpoint URL of the AP
 
 *Note: The test environment should be running a cronjob for `token-expirator` otherwise the `testTemporaryAccess` test will fail.*
 
-Create file `.env` with environment variables:
+Create file `.env` with environment variables or copy `.env.dist`:
```

Comment:
tady furt zmiňuješ `.env.dist`

Reply:
Jé, na to jsem zapomněl ... Je to fuč, díky.